tgui/packages/tgui/interfaces/SeedExtractor.js

```diff
@@ -0,0 +1,88 @@
+import { useBackend } from '../backend';
+import { Button, Section, Table, Box } from '../components';
+import { ComplexModal, modalOpen} from './common/ComplexModal';
+import { Window } from '../layouts';
+
+export const SeedExtractor = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    stored_seeds,
+    vend_number,
+  } = data;
+
+  return (
+    <Window resizable>
+      <ComplexModal/>
+      <Window.Content scrollable className="Layout__content--flexColumn">
+        <Section title="Stored Seeds">
+          <div className="CameraConsole__toolbarRight">
+            Will Vend:&nbsp;
+            <Button
```

Comment:
Can it be a dropdown instead?
I think the semantics of clicking it (i.e. "you can CHOOSE how much would be dispensed") would be clearer that way.
That, or maybe a number input box? (i don't have a particular preference between a dropdown and an input)

Reply:
im going to put in a number input box

Reply:
done